src/DCON/dcon_mod.jl

```diff
@@ -0,0 +1,26 @@
+module DconMod
+
+# All imports and includes for the DCON module
+using JPEC 
```

Comment:
This will cause problems. JPEC is a top-level module. DCON, a submodule, shouldn't import it. DCON either needs to be inside the JPEC module (some `include("DCON/dcon_mod.jl")` in `JPEC.jl`, or you need to make it its own package with its own Project.toml file (see https://github.com/OpenFUSIONToolkit/JPEC/issues/4#issuecomment-3145611875)